Q12.py

```diff
@@ -0,0 +1,45 @@
+class BankAccount:
+    def __init__(self, acc_name, acc_number, balance, pin):
+        self.acc_name = acc_name
+        self.acc_number=acc_number
+        self.balance=balance
```

Comment:
The space before and after `=` should be consistent. For above lines, you have specified space but here you haven't.